Library/Homebrew/os/mac/xcode.rb

```diff
@@ -254,9 +254,8 @@ def detect_version
         # CLT isn't a distinct entity pre-4.3, and pkgutil doesn't exist
         # at all on Tiger, so just count it as installed if Xcode is installed
         return MacOS::Xcode.version if MacOS::Xcode.installed? && Xcode::Version.new(MacOS::Xcode.version) < "3.0"
-
```

Comment:
I did not notice I had removed it. I had debug print statements there. I added it back.